docs/development/extending.md

```diff
@@ -343,15 +349,136 @@ collections/
             └── roles
 ```
 
-!!! tip
-    Only underscores (`_`) are allowed for filenames inside collections!  
-    Naming the file `cc-filter-plugins.py` will result in an error!
-
 Now, the filter can be used:
 
 ```yaml
 sorted_ip_list: "{{ ip_list | computacenter.utils.sort_ip }}"
 ```
 
-!!! tip
-    The usage of the FQCN for your Plugin is mandatory!
+### Dynamic inventory plugins
+
+Ansible can pull informations from different sources, like ServiceNow, Cisco etc. If your source is not covered with the integrated inventory plugins, you can create your own.
+
+For more informations take a look at [Ansible docs - Developing inventory plugin](https://docs.ansible.com/ansible/latest/dev_guide/developing_inventory.html){:target="_blank"}.
+
+**Key things to note:**
+
+* The DOCUMENTATION section is required and used by the plugin. Note how the options here reflect exactly the options we specified in the csv_inventory.yaml file in the previous step.
+* The NAME should exactly match the name of the plugin everywhere else.
+* For details on the imports and base classes/helpers.
+
+Documentaion -> Declare option that are needed in the plugin.
+Examples -> Example with parameter for a inventory file to run the script.
+Python Code -> Different methods like verify_file, parse and more.
```

Comment:
Was ist mit dieser Aufstellung gemeint? Ich hatte die Liste mit den drei Punkten bereits korrigiert für ein korrektes Rendering. Müssen Zeile 370 bis 372 auch in eine Liste? Kannst du hier noch mehr und besser Informationen einfügen?
Für ein Pfeil-Symbol kannst du `:material-arrow-right:` verwenden.

Reply:
done